src/routes/news/components/NewsList.js

```diff
@@ -0,0 +1,251 @@
+// @flow
+
+import * as React from 'react'
+import { View, ScrollView, Linking } from 'react-native'
+import { TFunction, withTranslation } from 'react-i18next'
+import {
+  LocalNewsModel,
+  TunewsModel
+} from '@integreat-app/integreat-api-client'
+import ContentNotFoundError from '../../../modules/error/ContentNotFoundError'
+import List from './List'
+import Failure from '../../../modules/error/components/Failure'
+import type { ThemeType } from '../../../modules/theme/constants/theme'
+import type { LanguageResourceCacheStateType } from '../../../modules/app/StateType'
+import type { NavigateToNewsParamsType } from '../../../modules/app/createNavigateToNews'
+import withTheme from '../../../modules/theme/hocs/withTheme'
+import SpaceBetween from '../../../modules/common/components/SpaceBetween'
+import ErrorCodes from '../../../modules/error/ErrorCodes'
+import NewsListItem from './NewsListItem'
+import headerImage from '../assets/tu-news-header-details-icon.svg'
+import styled from 'styled-components/native'
+import type { StyledComponent } from 'styled-components'
+import { TUNEWS } from '../containers/WithCustomNewsProvider'
+import { contentAlignment, contentDirection } from '../../../modules/i18n/contentDirection'
+import moment from 'moment'
+const tunewsWebsiteUrl = 'https://tunewsinternational.com'
+
+const Container: StyledComponent<{}, {}, *> = styled.View`
+  align-items: center;
+  margin-horizontal: 3%;
+  flex: 1;
+`
+
+const HeaderImageWrapper: StyledComponent<{}, {}, *> = styled.View`
+  width: 95%;
+  align-self: center;
+  align-items: flex-start;
+  margin-top: 19px;
+  border-radius: 5px;
+  background-color: rgba(2, 121, 166, 0.4);
+`
+
+const HeaderImage: StyledComponent<{}, {}, *> = styled.Image`
+  border-top-left-radius: 5px;
+  border-bottom-left-radius: 5px;
+`
+
+const Row: StyledComponent<{language: string}, ThemeType, *> = styled.View`
+  flex-direction: ${props => contentDirection(props.language)};
+  border-radius: 5px;
+  width: 95%;
+  align-self: center;
+  background-color: #0279a6;
```

Comment:
Please do not just use some colors but rather add them to the theme (or use similar colors already in the theme). Probably it would make sense to take a look at the webapp for this.